crates/uv-settings/src/lib.rs

```diff
@@ -206,3 +259,109 @@ pub enum Error {
     #[error("Failed to parse: `{0}`")]
     UvToml(String, #[source] toml::de::Error),
 }
+
+#[cfg(test)]
+mod test {
+    #[cfg(not(windows))]
+    use crate::locate_system_config_xdg;
+    #[cfg(windows)]
+    use crate::system_config_file;
+    #[cfg(windows)]
+    use uv_static::EnvVars;
+
+    use assert_fs::fixture::FixtureError;
+    use assert_fs::prelude::*;
+    use indoc::indoc;
+    use std::env;
+
+    #[test]
+    #[cfg(not(windows))]
+    fn test_locate_system_config_xdg() -> Result<(), FixtureError> {
+        // Write a `uv.toml` to a temporary directory.
+        let context = assert_fs::TempDir::new()?;
+        context.child("uv").child("uv.toml").write_str(indoc! {
+            r#"
+            [pip]
+            index-url = "https://test.pypi.org/simple"
+        "#,
+        })?;
+
+        let current_dir = env::current_dir().unwrap();
+
+        {
```

Comment:
Why is this in a block?

Reply:
To clarify the scope.

Reply:
The scope of changing the current directory? I think it's misleading because it's not like `set_current_dir` reverts when the scope exits.